app/settings/common.py

```diff
@@ -49,7 +49,7 @@
     },
     'loggers': {
         '': {
-            'level': 'DEBUG' if env.bool('DEBUG') else 'INFO',
+            'level': 'INFO' if env.bool('DEBUG') else 'INFO',
```

Comment:
Can this be just 'INFO' now?
```suggestion
            'level': 'INFO',
```

Reply:
yes, finding DEBUG too verbose, will make it INFO.